index.bs

```diff
@@ -1370,77 +880,818 @@ What configuration is the device in after it resets?
     constructor(sequence<USBIsochronousOutTransferPacket> packets);
     readonly attribute FrozenArray<USBIsochronousOutTransferPacket> packets;
   };
+
+  [Exposed=(DedicatedWorker,SharedWorker,Window), SecureContext]
+  interface USBDevice {
+    readonly attribute octet usbVersionMajor;
+    readonly attribute octet usbVersionMinor;
+    readonly attribute octet usbVersionSubminor;
+    readonly attribute octet deviceClass;
+    readonly attribute octet deviceSubclass;
+    readonly attribute octet deviceProtocol;
+    readonly attribute unsigned short vendorId;
+    readonly attribute unsigned short productId;
+    readonly attribute octet deviceVersionMajor;
+    readonly attribute octet deviceVersionMinor;
+    readonly attribute octet deviceVersionSubminor;
+    readonly attribute DOMString? manufacturerName;
+    readonly attribute DOMString? productName;
+    readonly attribute DOMString? serialNumber;
+    [SameObject] readonly attribute USBConfiguration? configuration;
```

Comment:
Since the value of `configuration` can change I don't think we can actually call this `[SameObject]` but the getter steps for the attribute are good enough to specify the behavior we want.

Reply:
Updated in https://github.com/WICG/webusb/pull/212/commits/1a96f493fc285da99d2b812d809c241388197beb.